frontend/src/js/ui-components/InputDate.tsx

```diff
@@ -53,13 +59,22 @@ const InputDate = forwardRef<HTMLInputElement, Props>(
       onChange,
       onCalendarSelect,
       onFocus,
+      onBlur,
+      onClick,
       ...props
     },
     ref,
   ) => {
     const datePickerRef = createRef<ReactDatePicker>();
```

Comment:
Just noticing this now, we should `useRef` to avoid recreating the ref on every render
```suggestion
    const datePickerRef = useRef<ReactDatePicker>();
```